sporkfish/searcher/move_ordering/composite_heuristic.py

```diff
@@ -8,26 +8,32 @@
 from .move_order_heuristic import MoveOrderHeuristic
 from .move_order_config import MoveOrderMode, MoveOrderConfig
 from .mvv_lva_heuristic import MvvLvaHeuristic
+from .history_heuristic import HistoryHeuristic
 
 
-class CompositeHeuristic(MvvLvaHeuristic, KillerMoveHeuristic, MoveOrderHeuristic):
+class CompositeHeuristic(
+    MvvLvaHeuristic, KillerMoveHeuristic, HistoryHeuristic, MoveOrderHeuristic
+):
     def __init__(
         self,
         board: Board,
         killer_moves: List[List[chess.Move]],
+        history_table: Dict[chess.Move, int],
         depth: int,
         move_order_config: MoveOrderConfig = MoveOrderConfig(),
     ) -> None:
         MvvLvaHeuristic.__init__(self, board)
         KillerMoveHeuristic.__init__(self, board, killer_moves, depth)
         MoveOrderHeuristic.__init__(self)
+        HistoryHeuristic.__init__(self, board, history_table)
```

Comment:
I suggest adding this in order of the multiple inheritance, i.e. history should go before move order heuristic. We might have issues with initalization due to ordering otherwise (I don't know know what exactly, but I had issues when I created this class).